admin_manual/configuration_database/db_conversion.rst

```diff
@@ -37,19 +31,17 @@ For example
 
 To successfully proceed with the conversion, you must type ``yes`` when prompted 
 with the question ``Continue with the conversion?``
-
 On success the converter will automatically configure the new database in your 
 ownCloud config ``config.php``.
 
 Unconvertible Tables
 --------------------
 
-If you updated your ownCloud installation there might exist old tables, which 
-are not used anymore. The converter will tell you which ones.
+If you updated your ownCloud installation old tables, which are not used anymore, might still exist. 
```

Comment:
I had to think when reading this, what are these "installation old tables"?
Explicitly putting "then" will resolve the sort-of ambiguity.
"If you updated your ownCloud installation then old tables,..."